tests/unit/test_user.py

```diff
@@ -26,6 +26,37 @@ def test_full_name(empty_test_db, create_admin_user):
     assert admin.full_name() == "first last"
 
 
+@pytest.mark.usefixtures("empty_test_db")
+class TestFindUsersWithoutAssignedRole:
+    def test_find_users_authorized_active(self, create_admin_user):
+        _ = create_admin_user()
+
+        found_user = UserModel.find_users_without_assigned_role()
+        assert found_user.first() is None
+
+    def test_find_users_authorized_inactive(self, create_admin_user):
+        user = create_admin_user()
+        user.archived = True
+        user.save_to_db()
```

Comment:
I wonder, I think the factories are setup to take keyword arguments.

I believe we can remove all this and replace it with `create_admin_user(archived=True)`

Well, I tested this theory out, and while it is setup to take keyword arguments. archived doesn't work as expected, but I think it would be ok to change that.

Reply:
I was going to say if you wanted to include this in this PR than that would be OK, but I think it would be best to create a separate issue for it. I will get an issue created after this PR is merged.

Reply:
Yeah, @ovsyankimo  and I already tried altering the fixture to do that.  
However, even when you do, `archived` stays as `False` because the `UserModel` has `self.archived=False` in the init method.
https://github.com/codeforpdx/dwellinglybackend/blob/b02379dffaa64b6352e93e1d45275bd81afa263b/models/user.py#L48-L51

I think that can be removed from the init, but I haven't gotten around to digging through the code to see if there was a reason for it to be there in the first place.  I have the feeling it is just a remnant from before we added `default=False` to the `archived` field.

Reply:
Yep, I remember when that was added. I think it can be removed without issue, if you want to do that in this pr, I think that would be fine, but if removing it causes other tests to fail then let’s do it separately.